universal-login-example/src/App.tsx

```diff
@@ -2,7 +2,8 @@ import React from 'react';
 import Web3 from 'web3';
 import {ULWeb3Provider} from '@universal-login/web3';
 
-const web3 = new Web3(new ULWeb3Provider(new Web3.providers.HttpProvider('https://rinkeby.infura.io')));
+const universalLogin = new ULWeb3Provider(new Web3.providers.HttpProvider('https://rinkeby.infura.io'));
```

Comment:
Infura has recently introduced API request limits. Here is our api key
`https://rinkeby.infura.io/v3/b3026fc5137a4bd18e5d5906ed49f77d`